On fsimx6, we are currenly using a 3.0.35 kernel that has Vivante 3D support
at the same level than 3.10.17 kernel. Therefore we need to use new imx-lib
from 3.10.17. But the PXP device is still the old one. This patch downgrades
the PXP library to work with 3.0.35 variant again.

diff -Naur imx-lib-3.10.17-1.0.0-orig/pxp/pxp_lib.c imx-lib-3.10.17-1.0.0-new/pxp/pxp_lib.c
--- imx-lib-3.10.17-1.0.0-orig/pxp/pxp_lib.c	2014-01-16 15:27:12.000000000 +0100
+++ imx-lib-3.10.17-1.0.0-new/pxp/pxp_lib.c	2015-12-21 15:48:01.000000000 +0100
@@ -71,21 +71,21 @@
 	if (ret < 0)
 		return -1;
 
-	pxp_chan->handle = cid;
+	pxp_chan->chan_id = cid;
 	return 0;
 }
 
 void pxp_release_channel(pxp_chan_handle_t *pxp_chan)
 {
-	ioctl(fd, PXP_IOC_PUT_CHAN, &(pxp_chan->handle));
+	ioctl(fd, PXP_IOC_PUT_CHAN, &(pxp_chan->chan_id));
 }
 
 int pxp_config_channel(pxp_chan_handle_t *pxp_chan, struct pxp_config_data *pxp_conf)
 {
 	int ret = 0;
 
-	pxp_conf->handle = pxp_chan->handle;
-	dbg(DBG_INFO, "channel handle %d\n\n", pxp_conf->handle);
+	pxp_conf->chan_id = pxp_chan->chan_id;
+	dbg(DBG_INFO, "channel id %d\n\n", pxp_conf->chan_id);
 
 	ret = ioctl(fd, PXP_IOC_CONFIG_CHAN, pxp_conf);
 
@@ -96,7 +96,7 @@
 {
 	int ret = 0;
 
-	ret = ioctl(fd, PXP_IOC_START_CHAN, &(pxp_chan->handle));
+	ret = ioctl(fd, PXP_IOC_START_CHAN, &(pxp_chan->chan_id));
 
 	return ret;
 }
